### DISCUSSION TITLE
Custom code changes detected. Aborting update. Any advice?

### DISCUSSION THREAD

**Discussion by yamlv** [chatwoot/chatwoot#8979] [Self hosted]
Hi, i have chatwoot v3.5.2, ubuntu 22.02, manual install, everything works fine. 

I noticed today i can upgrade to. 3.6 version i run this command as root:

sudo cwctl --upgrade

And got this:

Checking for cwctl updates...
Your cwctl is up to date
Upgrading Chatwoot to v3.6.0
config/initializers/rack_attack.rb: needs update
Custom code changes detected. Aborting update.
Please proceed to update manually.

Any suggestion how to force updgrade? how to upgrade manually ? or hot to fix custom code changes ? i dont even remember if i changed anything.. is there any log which can show custom changes if any was ? 

Thank you, appreciate it.

**Comment by sojan-official** [comment]
Hey Yamiv 

You can navigate to the chatwoot directory and do `git stash` , which will preserve the changes you made into a stash, after which you can continue with the upgrade. 
You can also do `git diff` to see the difference 

cheers
Sojan
